libraries/core_libs/consensus/src/vote_manager/vote_manager.cpp

```diff
@@ -30,29 +31,12 @@ VoteManager::~VoteManager() { daemon_->join(); }
 void VoteManager::setNetwork(std::weak_ptr<Network> network) { network_ = std::move(network); }
 
 void VoteManager::retreieveVotes_() {
-  LOG(log_si_) << "Retrieve unverified votes from DB";
-  auto unverified_votes = db_->getUnverifiedVotes();
-  for (auto const& v : unverified_votes) {
-    auto pbft_round = v->getRound();
-    auto hash = v->getHash();
-    {
-      UniqueLock lock(unverified_votes_access_);
-      if (unverified_votes_.count(pbft_round)) {
-        unverified_votes_[pbft_round][hash] = v;
-      } else {
-        std::unordered_map<vote_hash_t, std::shared_ptr<Vote>> votes{std::make_pair(hash, v)};
-        unverified_votes_[pbft_round] = std::move(votes);
-      }
-    }
-    LOG(log_dg_) << "Retrieved unverified vote " << *v;
-  }
```

Comment:
If don't retrieve those unverified votes, will need wait peers sending again?

Reply:
yes, unverified votes are note saved as they could be irrelevant after restart e.g. network progress could be few pbft block meanwhile